Dockerfile

```
@@ -4,8 +4,10 @@ WORKDIR /app
 
 # copy repo contents
 COPY setup.py README.md ./
-COPY optimade ./optimade
-RUN pip install -e .[server]
+COPY optimade-core ./optimade-core
+COPY optimade-server ./optimade-server
+RUN pip install -e optimade-core/
+RUN pip install -e optimade-server/
```

This should fix the docker validator failures?

```suggestion
COPY optimade-core ./optimade-core
COPY optimade-server ./optimade-server
COPY optimade-validator ./optimade-validator
RUN pip install -e optimade-core/
RUN pip install -e optimade-server/
RUN pip install -e optimade-validator/
```

---

Let's try updating the docker to fix the validator